internal/controller/controllers/installenv_controller_test.go

```diff
@@ -284,15 +284,17 @@ var _ = Describe("installEnv reconcile", func() {
 		Expect(conditionsv1.FindStatusCondition(installEnvImage.Status.Conditions, adiiov1alpha1.ImageCreatedCondition).Status).To(Equal(corev1.ConditionUnknown))
 	})
 
-	It("create image with proxy configuration and ntp sources", func() {
+	It("create image with proxy configuration, ntp sources and ignition config override", func() {
```

Comment:
What about failure case?
Consider splitting the tests, it's a unit test after all

Reply:
Done